binzx/otomi

```diff
@@ -41,37 +41,73 @@ silent() {
     "$@" &>/dev/null
   fi
 }
+
+docker_tag_exists() {
+  curl -sflL https://index.docker.io/v1/repositories/$1/tags/$2 &>/dev/null
+}
+
 # shellcheck disable=SC2155
 readonly base_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
 
-if [[ $base_dir == *"otomi-core"* ]] || [[ $(pwd) == *"otomi-core"* ]]; then
+findUp() {
+  path=$1
+  while [[ "$path" != "" && ! -e "$path/$2" ]]; do
+    path=${path%/*}
+  done
+  echo "$path"
+}
```

Comment:
not needed

Reply:
I disagree. I want to be able to run otomi in dev mode, even if my terminal is not in the root directory of otomi-core.